doc/swagger/swagger.yaml

```diff
@@ -27,6 +27,36 @@ tags:
     description: Available schemas under a project for validating published messages payload
 paths:
 
+  /projects/{PROJECT}/schemas:
+    get:
+      summary: Retrieve all schemas under a project
+      description: |
+        Retrieve all schemas under a project
+      parameters:
```

Comment:
Is there also an optional schema name url var that must be declared here?

Reply:
There was indeed a reference to the url name parameter of the schema in the handler which was wrong.Updated the handler and removed any redundant code.